_posts/courses/earth-analytics-python/01-science-toolbox/2018-02-05-science-toolbox-landing.md

```diff
@@ -89,7 +89,7 @@ You will be behind if these things are not setup / complete before next week.
 
 You have two options to complete activities assigned in this course. 
 
-1. You can chose to either work on the assignments locally on your computer. 
+1. You can choose to either work on the assignments locally on your computer. 
 
 <a href= "https://www.earthdatascience.org/workshops/setup-earth-analytics-python/">Setup Earth Analytics Python Environment on Your Laptop</a>
 
```

Comment:
Is the change on this lesson that is missing? @jlpalomino i'm just not sure what to look for. a line number would be helpful

Reply:
No - Line 203 - I updated the image for the homework 1 plot with a new title. It looks like you resolved it with: https://github.com/earthlab/earthlab.github.io/pull/619/commits/6972f357343e3a62d97cac3e4cf8d40efa2cdb66